pkg/datapath/linux/ipsecrps/ipsec_rps_linux.go

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package ipsecrps
+
+import (
+	"errors"
+	"fmt"
+	"runtime"
+	"strconv"
+
+	"github.com/cilium/hive/cell"
+	"github.com/spf13/pflag"
+
+	dpcfgdef "github.com/cilium/cilium/pkg/datapath/linux/config/defines"
+	"github.com/cilium/cilium/pkg/datapath/tunnel"
+	"github.com/cilium/cilium/pkg/datapath/xdp"
+	"github.com/cilium/cilium/pkg/maps/cpumap"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+// userFlags represents the options that the user provided to the daemon
+// to control the IPSec RPS configuration
+type userFlags struct {
+	EnableIpsecAcceleration bool
+}
+
+func (u userFlags) Flags(flags *pflag.FlagSet) {
+	flags.Bool("enable-ipsec-acceleration", u.EnableIpsecAcceleration, "Enable acceleration for IPSec using RPS.")
```

Comment:
Similarly to above, probably worth adding an `(experimental)` into the description here so users don't turn this on without consulting the docs

Reply:
Done!